modules/subscriptions/views.php

```diff
@@ -5,7 +5,13 @@ class Jetpack_Subscriptions_Widget extends WP_Widget {
 	/**
 	 * @var array When printing the submit button, what tags are allowed
 	 */
-	static $allowed_html_tags_for_submit_button = array( 'br' => array() );
+	public static $allowed_html_tags_for_submit_button = array(
+		'br'     => array(),
+		's'      => array(),
```

Comment:
What's the "s" tag for?

Reply:
xhtml strikeout? — it might be deprecated in html5 so prolly best not include.

Reply:
I believe the editor was inserting this when using strikethrough, but I'll double check.

Reply:
Yes, using strikethrough in a RichText component, the editor adds the `<s>` tag.